cloud_Azure/terraform/module/service_principal.tf

```diff
@@ -0,0 +1,12 @@
+data "azuread_client_config" "current" {}
+
+# Creates Service Principal for pre-existing "Kentik NSG Flow Exporter" app, so the app can access flow logs in Azure cloud
+resource "azuread_service_principal" "kentik_nsg_flow_exporter" {
+  application_id               = "a20ce222-63c0-46db-86d5-58551eeee89f" # pre-existing app ID
```

Comment:
This should probably be a module variable

Reply:
Declaring as variable with default  value of a20ce222-63c0-46db-86d5-58551eeee89f